operator/pkg/reconciliation/reconcile_racks.go

```diff
@@ -75,23 +75,17 @@ func (rc *ReconciliationContext) CalculateRackInformation() error {
 		return fmt.Errorf("assertion failed! rackCount should not possibly be zero here")
 	}
 
-	// nodes_per_rack = total_size / rack_count + 1 if rack_index < remainder
-
-	nodesPerRack, extraNodes := nodeCount/rackCount, nodeCount%rackCount
 	seedsPerRack, extraSeeds := seedCount/rackCount, seedCount%rackCount
```

Comment:
you could use

```suggestion
	rackSeedCounts := api.SplitRacks(seedCount, rackCount)
```

with some more changes below here too

Reply:
ah good idea